src/dynamic_fmt.c

```diff
@@ -3495,17 +3495,17 @@ inline static void __append2_string(DYNA_OMP_PARAMSm unsigned char *Str, unsigne
 	}
 }
 
-void DynamicFunc__setmode_unicodeBE(DYNA_OMP_PARAMS) // DYNA_OMP_PARAMS not used. We use omp_thread_num() instead.
+void DynamicFunc__setmode_unicodeBE(DYNA_OMP_PARAMS) // DYNA_OMP_PARAMS not used. We use threadshread_num() instead.
 {
 	md5_unicode_convert_set(2,tid);
 }
 
-void DynamicFunc__setmode_unicode(DYNA_OMP_PARAMS) // DYNA_OMP_PARAMS not used. We use omp_thread_num() instead.
+void DynamicFunc__setmode_unicode(DYNA_OMP_PARAMS) // DYNA_OMP_PARAMS not used. We use threadshread_num() instead.
 {
 	md5_unicode_convert_set(1,tid);
 }
 
-void DynamicFunc__setmode_normal (DYNA_OMP_PARAMS) // DYNA_OMP_PARAMS not used. We use omp_thread_num() instead.
+void DynamicFunc__setmode_normal (DYNA_OMP_PARAMS) // DYNA_OMP_PARAMS not used. We use threadshread_num() instead.
```

Comment:
Ouch.

Reply:
Oops, my bad. I could not double-check my results earlier this morning.